chsdi/views/features.py

```diff
@@ -401,15 +401,21 @@ def _get_features_for_filters(params, models, maxFeatures=None, where=None):
             if where is not None or geomFilter is not None:
                 # TODO remove layer specific code
                 if model.__bodId__ == 'ch.swisstopo.zeitreihen':
-                    counter = 0
-                    bgdi_order = 0
-                    for feature in query:
-                        counter += 1
-                        if counter > 1:
-                            if bgdi_order < feature.bgdi_order:
-                                continue
-                        bgdi_order = feature.bgdi_order
-                        yield feature
+                    # standard identify show first bgdi_order only
+                    if maxFeatures == 201:
```

Comment:
This if could be put togehter with the previous if:

`if model.__bodId__ == 'ch.swisstopo.zeitreihen' and maxFeatures == 201:`

201 is magic number. Would prefer to have constant defined at top of file MAX_FEATURES = 201 that is then used verywhere.